backend/prisma/schema.prisma

```diff
@@ -25,12 +25,20 @@ model User {
 }
 
 model Event {
-  eventId   String   @id
-  owner     User     @relation(fields: [ownerId], references: [userId])
+  eventId   String    @id @default(uuid())
+  owner     User      @relation(fields: [ownerId], references: [userId])
   ownerId   String
-  startTime DateTime @db.Timestamptz()
-  endTime   DateTime @db.Timestamptz()
+  startTime DateTime  @db.Timestamptz()
+  endTime   DateTime? @db.Timestamptz()
   type      String
+  foodType  String?
+  amount    Float?
+  note      String?
+  unit      String?
+  // We will have to link this to baby eventually,
+  //  but we first have to figure out how to handle users with multiple babies
```

Comment:
For this, is there a reason that we make an event related to Baby, not User? I think we can find the User easily because Baby has a parentId field. Also, I am thinking of making Plan related to Event. For example, a plan will have a list of Event, and each event can be either nap, food, or etc. What do you think?

Reply:
I don't think the plan should be related to the events, because events are things that actually happened and plans have planned events that have not occurred. But, I'm not sure how you are visualizing the plan so I don't know. In terms of the events, they are related to the user right now, not baby. We would need to relate them to the baby eventually because one user can have multiple babies that they are keeping track of, so if the events are only related to the user, you don't know what baby they belong to.

Reply:
I see. I was wondering if the events have to be related to users at all. I guess we can change that later. Also, the reason that I was thinking of making plan related to event was because plan will consist of events as well. Each event in plan will also have a start time, end time, type, etc. But you are right that plan's event and log should be in a separate table.